model/patch_lifecycle.go

```diff
@@ -546,13 +581,7 @@ func MakeMergePatchFromExisting(existingPatchID string) (*patch.Patch, error) {
 	if projectRef == nil {
 		return nil, errors.Errorf("no project '%s' exists", existingPatch.Project)
 	}
-	if !projectRef.Enabled {
-		return nil, errors.Errorf("project '%s' is disabled", existingPatch.Project)
-	}
-	if projectRef.PatchingDisabled {
```

Comment:
I think each of these errors is helpful. Can the new function instead return the specific error?

Reply:
done